test/plugin/constants.rb

```diff
@@ -510,6 +509,10 @@ module Constants
       )
     )
   ).freeze
+  # Used in fallback tests.
```

Comment:
Want to keep it with the other `COMPUTE_PARAMS_*` constants?

Reply:
Done.